roles/ceph-common/tasks/checks/check_ntp_atomic.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: check ntp installation on atomic
+  command: rpm -q chrony
+  register: ntp_pkg_query
+  ignore_errors: true
+  changed_when: false
+  when: ansible_os_family == 'RedHat'
```

Comment:
The is_atomic is checked before including ntp_atomic.yml->check_ntp_atomic.yml so that should be covered there. I wanted to avoid duplicating the is_atomic register that is already done in the role's .../docker/main.yml. I thought about passing in the is_atomic variable but ultimately opted for this instead. The when check here is in case anybody wants to include this file individually.

If you'd like, I could see about just using is_atomic alone either by requiring callers to pass it in, or registering it here as well despite it being duplicated.